kazungusafari/AUTOMART
# 166270947 Refactor code as per Cogrammar review

What does this PR do?
Refactor code as per review

Description of Task to completed?
Resize modals and make them proportional to the content inside
the modal.s

How should this be manually tested?
After cloning this repository, go to my-sale-ads.html and click update price link,the modal should now be proportional

What is relevant pivotal tracker story?
#166270947
